web-console/conceptual-designs/topology.md

```diff
@@ -0,0 +1,15 @@
+# Topology View
+- A topology overview page will enable administrators and developers to easily visualize their applications, to mentally organize application components, and to get a better insight into the health and connections of those components.
+
+## Use Cases
+1. As a system administrator, I want to:
+  - See how the components of my project(or projects) are connected together.
+  - Verify that the components of my project are all present and functioning correctly.
+  - See which parts of my project are connected to external components.
+1. As a developer, I want to:
```

Comment:
Should this be a 2? This syntax may work with markdown, it's just not coming through when I view it but maybe once merged it will. Just checking :)